CMakeLists.txt

```diff
@@ -116,6 +117,9 @@ if (DNSSD_FOUND)
 		set(ASEBA_ZEROCONF_ADDITIONAL_LIBRARIES ${AVAHI_CLIENT_LIBRARIES} ${AVAHI_COMMON_LIBRARIES} ${DNSSD_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 	else()
 		set(ASEBA_ZEROCONF_ADDITIONAL_LIBRARIES ${DNSSD_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+		if (WIN32)
+			set(ASEBA_ZEROCONF_ADDITIONAL_LIBRARIES ${ASEBA_ZEROCONF_ADDITIONAL_LIBRARIES} -lws2_32)
```

Comment:
I think that this line should go to the CMakeLists.txt specific to Zeroconf that sets `${ASEBA_ZEROCONF_ADDITIONAL_LIBRARIES}`, isn't it?

Reply:
OK, I will try that

Reply:
In fact, `${ASEBA_ZEROCONF_ADDITIONAL_LIBRARIES}` is set right here in the root CMakeLists.txt, just before the proposed addition. So this seems to be the appropriate place after all.

Reply:
Ok, right.